pack/ramp-oss-cluster.yml

```diff
@@ -0,0 +1,52 @@
+display_name: RediSearch 2 OSS cluster
+capability_name: Search and query
+author: RedisLabs
+email: meir@redislabs.com
+description: High performance search index on top of redis (with oss clustering)
+homepage: 'http://redisearch.io'
+license: Redis Source Available License 2.0 (RSALv2) or the Server Side Public License v1 (SSPLv1)
+command_line_args: PARTITIONS AUTO
+min_redis_version: '6.0.7'
+min_redis_pack_version: '6.0.20'
```

Comment:
Should it be different than 6.0.8 as in
https://github.com/RediSearch/RediSearch/blob/2.6/coord/pack/ramp.yml#L9

Reply:
OK. Done.